server/study_groups/methods.js

```diff
@@ -454,3 +454,59 @@ Meteor.methods({
     return true;
   }
 });
+
+Meteor.methods({
+  transferStudyGroup: function(data) {
+    check(data, {
+      studyGroupId: String,
+      newOwnerId: String,
+      studyGroupTitle: String,
+      studyGroupSlug: String
+    });
+
+    if (!this.userId) {
+      throw new Meteor.Error('Members.methods.editStatus.not-logged-in', 'Must be logged in to transfer study group.');
+    }
+
+    const actor = Meteor.user()
+    if (!actor || !Roles.userIsInRole(actor, ['owner','admin', 'moderator', 'member'], data.studyGroupId )) {
```

Comment:
Please modify this.
Only owner should be able transfer the membership. 
i.e. `!Roles.userIsInRole(actor, ['owner'], data.studyGroupId )`